pkg/webui/console/containers/users-table/index.js

```diff
@@ -39,126 +52,254 @@ import {
 
 import style from './users-table.styl'
 
-@connect(state => ({
-  currentUserId: selectUserId(state),
-}))
-export default class UsersTable extends Component {
-  static propTypes = {
-    currentUserId: PropTypes.string.isRequired,
-    pageSize: PropTypes.number.isRequired,
-  }
-
-  constructor(props) {
-    super(props)
-    this.headers = [
-      {
-        name: 'ids.user_id',
-        displayName: sharedMessages.id,
-        width: 28,
-        sortable: true,
-        sortKey: 'user_id',
-        render: ids => {
-          const userId = getUserId({ ids })
-          if (userId === props.currentUserId) {
-            return (
-              <span>
-                {userId}{' '}
-                <Message className={style.hint} content={sharedMessages.currentUserIndicator} />
-              </span>
-            )
-          }
-          return userId
+const m = defineMessages({
+  invite: 'Invite user',
+  revokeInvitation: 'Revoke this invitation',
+  sentAt: 'Sent',
+  revokeSuccess: 'Invite removed successfully',
+  revokeError: 'There was an error and the invite could not be revoked',
+})
+
+const USERS_TAB = 'users'
+const INVITATIONS_TAB = 'invitations'
+const tabs = [
+  {
+    title: 'Users',
+    name: 'users',
+  },
+  {
+    title: 'Invitations',
```

Comment:
```suggestion
    title: sharedMessages.userInvitations,
```

`User invitations`, please add this message.